packages/tail-kit/types/components/modal/modal.d.ts

```diff
@@ -25,19 +27,26 @@ export declare type ModalProps = {
   onRequestClose?: () => void
   /** custom actions button instead of OK and Cancel */
   actions?: React.ReactNode
+  /** Show dividers on top and bottom of Modal children  */
+  dividers?: boolean
```

Comment:
This doesn't seem like a good prop. Its OK for now, but I think while theming we have to remove it. Does any other library provide this kind of style props?

Reply:
material-ui -> https://next.material-ui.com/components/dialogs/#customized-dialogs
https://next.material-ui.com/api/dialog-content/#props